### PR TITLE
added some challenge CSS

### PR DESCRIPTION
**Pull request by katieannemills**
pulled the worthy bits out of https://github.com/swcarpentry/sql-novice-survey/pull/7 and inserted them here, per @gvwilson 's suggestion.
- adds a `.challenge` CSS class (just like the learning goals, but different color)
- removes some dead CSS from styles long past.

### CODE REVIEW

File: css/swc.css [hunk 1]
```diff
@@ -18,11 +18,6 @@ div.chapter h2 {
     font-style: italic;
 }
 
-/* Objectives and key points */
-.objectives, .keypoints {
```
Comment (rgaiacs) [inline css/swc.css:22]: Why did you remove `.objectives`? We drop `.keypoints` but `.objectives` wasn't drop (see https://raw.githubusercontent.com/swcarpentry/lesson-template/gh-pages/01-one.md).

Comment (katieannemills) [inline css/swc.css:22]: The deleted instance of `.objectives` was redundant; it is repeated at line 116.
